internal/k8s/container.go

```diff
@@ -99,6 +101,37 @@ func IsUnschedulable(pod v1.PodStatus) (bool, string) {
 	return false, ""
 }
 
+// Kubernetes has a bug where the image ref in the container status can be wrong.
```

Comment:
mayyybe worth mentioning
> Although it doesn't really make a difference as both image tags share the same digest it is technically wrong.

(i.e. "though this bug does NOT result in the container running unexpected code"). but also, 🤷‍♀

Reply:
done